### PR TITLE
Show and Tell: Open videos and pictures on click (editor)

### LINKED ISSUE #565: [Idea]: Show and tell admin: Open videos and pictures on click
### Describe your idea / enhancement

It would help if you could open video links and pictures in the review/edit view of show and tell entries. This probably also applied to the creation of posts. They could either open in a modal or just in a new tab.

### How would this idea / enhancement be beneficial?

Make reviews easier and allow people to verify they entered the correct links/uploaded the correct pictures when submitting.

### PR DESCRIPTION
**Pull request by ACA30**
## Describe your changes

Resolves #565. Allows quick preview of images (in a modal) and videos (open in new tab) on the Show & Tell editor form.

image

## Notes for testing your change

N/A

### CODE REVIEW

**Comment by ACA30** [comment]
I missed the ModalDialog component somehow???

**Comment by bl4nkfce** [comment]
Hey,

For UX and a11y purposes I'd suggest using the `button` element instead of a `div` when triggering the image preview.
It can be a separate "preview" button or you can swap the div for it.

By using the actual `button` element we get all the keyboard support that's expected from an interactive element.
Also, we should set some meaningful `title=` attribute for screen reader users.

File: apps/website/src/components/shared/form/ImageUploadAttachment.tsx [hunk 1]
```diff
@@ -32,7 +34,10 @@ export function ImageUploadAttachment({
   return (
     <div className="flex flex-row gap-5 rounded-lg bg-white p-2 px-4 shadow-lg">
       <div className="py-2">
-        <div className="relative h-32 w-32 overflow-hidden rounded-lg bg-gray-200">
+        <div
+          className="relative h-32 w-32 cursor-pointer overflow-hidden rounded-lg bg-gray-200"
+          onClick={onClick}
+        >
```
Comment (pjeweb) [inline apps/website/src/components/shared/form/ImageUploadAttachment.tsx:40]: Agreeing with @bl4nkfce that this should be a <button>

File: apps/website/src/components/shared/form/VideoLinksField.tsx
Comment (pjeweb) [inline apps/website/src/components/shared/form/VideoLinksField.tsx:1]: oopsie

**Review by pjeweb — CHANGES_REQUESTED** [review]
This is great. Thank you for working on this!

**Comment by MattIPv4** [comment]
👋 @ACA30 any update on this?

**Comment by MattIPv4** [comment]
No progress, closing.

**Pull request closed without merging**